lib/SQS.js

```diff
@@ -370,10 +356,10 @@ function connectToQueue(queueUrl, callback) {
 				Name: queueUrl.split("/").pop(),
 				URL: queueUrl,
 				Created: new Date().toISOString(),
-				DelaySeconds: 0,
+				DelaySeconds: 10,
```

Comment:
@britebit, thanks for the PR. Is there a reason you've changed the default `DelaySeconds` and `ReceiveMessageWaitTimeSeconds`? Maybe a relic from testing?

It would also be nice to have a test for this new condition.Thanks again! 😄